addons/stock/report/stock_report_views.xml

```diff
@@ -127,15 +127,6 @@
             <field name="binding_model_id" ref="model_stock_quant_package"/>
             <field name="binding_type">report</field>
         </record>
-        <record id="label_product_packaging" model="ir.actions.report">
-            <field name="name">Product Packaging (ZPL)</field>
```

Comment:
Not too used to the ZPL label usage, but we may need some kind of replacement for that one.